sanket/javascript/day17/assignment3/sanketA3_Q2.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+  </head>
+  <body>
+    no1: <input type="text" id="no1" />
+    <button onclick="cal()">calculate</button>
+
+    <script>
+      function cal() {
+        let n = parseInt(document.getElementById("no1").value);
```

Comment:
change variable name

Reply:
ma'am thank you for you reply i'll change the variable name can you please specify the variable naming standard you want me to apply